Assets/MirageXR/Common/Scripts/Managers/PointCloudManager.cs

```diff
@@ -12,6 +12,10 @@ public class PointCloudManager : MonoBehaviour
 
     public async Task<bool> InitializationAsync()
     {
+#if !UNITY_ANDROID && !UNITY_IOS
+        return;
+#endif
+
```

Comment:
return; throws and error as the method is expecting a bool. Change to return false;